### PR TITLE
BETA: Register evil3301.is-a.dev

### PR DESCRIPTION
**Pull request by Evil3301**
Added `evil3301.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/evil1337.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Evil3301",
+        "email": "idkanything011@gmail.com"
+    },
+    "record": {
+        "CNAME": "1"
```
Comment (wdhdev) [inline domains/evil1337.json:7]: Please use a valid record.

File: domains/evil3301.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Evil3301",
+        "email": "idkanything011@gmail.com"
+    },
+    "record": {
+        "URL": "https://evil3301.github.io/"
```
Comment (wdhdev) [inline domains/evil3301.json:7]:
```suggestion
        "CNAME": "evil3301.github.io"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/evil1337.json (hunk 1 above)
Comment (Evil3301) [reply domains/evil1337.json:7]: Idk what is record pls guide me

Comment (wdhdev) [reply domains/evil1337.json:7]: https://www.is-a.dev/docs

**Pull request closed without merging**